src/index.html

```diff
@@ -7,12 +7,66 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+
     <link
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
Browsers do not directly interpret SCSS files. Ensure that the 'href' attribute points to a compiled CSS file instead of an SCSS file.